static/js/biomedical/landing/highlights_section.tsx

```diff
@@ -85,18 +137,24 @@ function formatNumber(number: number, locale: string): string {
 export function HighlightsSection(props: HighlightsSectionProps): JSX.Element {
   return (
     <HighlightsContainer className="container">
-      {props.config.map((callout, index) => {
-        const value =
-          typeof callout.value === "number"
-            ? formatNumber(callout.value, props.locale)
-            : callout.value;
-        return (
-          <Label key={index}>
-            {callout.label}
-            <Highlight>{value}</Highlight>
-          </Label>
-        );
-      })}
+      <NumbersContainer>
+        {props.config.map((callout, index) => {
+          const value =
+            typeof callout.value === "number"
```

Comment:
Remove this logic since the `string` path is no longer used

Reply:
removed check for number